hack/make.ps1

```diff
@@ -363,6 +379,10 @@ Try {
     $root = $(Split-Path $MyInvocation.MyCommand.Definition -Parent | Split-Path -Parent)
     Push-Location $root
 
+    # Ensure the bundles directory exists
+    $bundlesDir = $root + "\bundles"
+    New-Item -Force $bundlesDir -ItemType Directory | Out-Null
```

Comment:
Could this use a relative path?

```powershell
New-Item -Force ".\bundles" -ItemType Directory | Out-Null
```

Reply:
That would mean we will need to set this in other places anyways where we cannot use a relative path[look at the function `Run-IntegrationTests` for example]. And it would also hide the intent that it is exactly the same directory that is being referred to in all of those places. The way it is used now, there are no dependencies that would need to be considered by someone modifying this code in the future. So I suggest we keep it as-is.

Reply:
Ah, yes, I see; it's not a "local" variable, but used globally. Wondering if we could somehow set it at the top; is there a convention to distinguish "local" from "global" variables in PowerShell? (I see other variables use an Uppercase) (or perhaps even a `const`?)

Reply:
This variable is in global scope and one can indeed distinguish locals and globals. Powershell does have constants but one cannot make a value constant after definition. One can however make it "Read Only" anytime - which is what I did.